ecs-cli/modules/cli/compose/entity/service/service.go

```diff
@@ -239,7 +239,10 @@ func (s *Service) Up() error {
 	deploymentConfig := s.DeploymentConfig()
 	// if the task definitions were different, updateService with new task definition
 	// this creates a deployment in ECS and slowly takes down the containers with old ones and starts new ones
-	err = s.Context().ECSClient.UpdateService(ecsServiceName, newTaskDefinitionId, newCount, deploymentConfig)
+
+	networkConfig := composeutils.ConvertToEcsNetworkConfiguration(s.projectContext.ECSParams)
```

Comment:
Unimportant nit: `ConvertToEcsNetworkConfiguration` should be `ConvertToECSNetworkConfiguration` according to the golang naming rules/suggestions.